samples/Samples.HttpMessageHandler/Program.cs

```diff
@@ -19,9 +19,6 @@ public static class Program
 
         private static string Url;
 
-#if NETFRAMEWORK
```

Comment:
Removing since all of the domain-neutral testing will be done in the new `Samples.MultiDomainHost.Runner` app